test/register_cleanup.go

```diff
@@ -0,0 +1,12 @@
+// +build go1.14
```

Comment:
I'm not completely sure whether it's worth using build tags just to make this work but it is annoying how long the fix tests to run when doing development, so I think it's worth considering.